nerfstudio/data/pixel_samplers.py

```diff
@@ -392,7 +392,7 @@ def __init__(self, config: PairPixelSamplerConfig, **kwargs) -> None:
     # overrides base method
     def sample_method(  # pylint: disable=no-self-use
         self,
-        batch_size: int,
+        batch_size: Optional[int],
```

Comment:
tanku for this fix!

is this param really optional tho?  in the superclass it is not optional.  it's a little confusing because the pixel sampler seems to have a side API for configuring batch size even tho notionally the batch size config can be passed in via a different flag